README.md

```diff
@@ -43,11 +55,11 @@ members or found helpful for managing open source projects and offices.
 - [MeasureOSS](https://github.com/MeasureOSS/Measure) - A contributor relationship management system
 - [GrimoireLab](https://chaoss.github.io/grimoirelab/) - Software development analytics platform supporting more than 30 different data sources, part of CHAOSS Software project from The Linux Foundation
 - [Starfish](https://github.com/indeedeng/starfish) - A tool to identify GitHub contributions within a specified window of time. 
+-[Project Portal](https://github.com/SAP/project-portal-for-innersource) - Lists all InnerSource projects of a company in an interactive and easy to use way. Can be used as a template for implementing the "InnerSource portal" pattern by the InnerSource Commons community.
```

Comment:
RE: our voice chat on the work day, @ashleywolf clarified that while this came out of innersource, this project can be used for any repos.